dev/com.ibm.websphere.appserver.features/visibility/public/beanValidation-3.0/resources/l10n/com.ibm.websphere.appserver.beanValidation-3.0.properties

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=The Bean Validation 3.0 specification provides an annotation based model for validating \
```

Comment:
change
annotation based
to
annotation-based

Reply:
I've made the requested update.  The file has been renamed to:
com.ibm.websphere.appserver.features/visibility/public/beanValidation-3.0/resources/l10n/io.openliberty.beanValidation-3.0.properties